### PR TITLE
Expose kbfssync.Semaphore.Count() method

### PR DESCRIPTION
**Pull request by akalin-keybase**
Use it in the diskLimiter implementations.

### CODE REVIEW

File: libkbfs/backpressure_disk_limiter.go [hunk 1]
```diff
@@ -120,20 +111,18 @@ func (s backpressureDiskLimiter) beforeBlockPut(
 	log logger.Logger) (int64, error) {
 	if blockBytes == 0 {
 		// Better to return an error than to panic in Acquire.
-		//
-		// TODO: Return current semaphore count.
-		return 0, errors.New("beforeBlockPut called with 0 blockBytes")
+		return s.s.Count(), errors.New(
```
Comment (akalin-keybase) [inline libkbfs/backpressure_disk_limiter.go]: fixed log here

File: libkbfs/backpressure_disk_limiter.go [hunk 2]
```diff
@@ -120,20 +111,18 @@ func (s backpressureDiskLimiter) beforeBlockPut(
...
+			"beforeBlockPut called with 0 blockBytes")
 	}
 
 	delay := s.getDelay()
 	if delay > 0 {
-		log.CDebugf(ctx, "Delaying block put of %d bytes by %d s",
-			delay.Seconds())
+		log.CDebugf(ctx, "Delaying block put of %d bytes by %f s",
```
Comment (akalin-keybase) [inline libkbfs/backpressure_disk_limiter.go]: Fixed log here

File: libkbfs/config_local.go [hunk 3]
```diff
@@ -924,13 +924,13 @@ func (c *ConfigLocal) EnableJournaling(
 	// the maximum duration.
 	const backpressureMaxThreshold = journalDiskLimit * 19 / 20
 
-	diskLimitSemaphore := newBackpressureDiskLimiter(
+	bdl := newBackpressureDiskLimiter(
```
Comment (akalin-keybase) [inline libkbfs/config_local.go]: fixed name here

File: kbfssync/semaphore.go [hunk 4]
```diff
@@ -30,9 +30,9 @@ func NewSemaphore() *Semaphore {
 	}
 }
 
-// countForTest returns the current resource count. It should be used
-// only for testing.
-func (s *Semaphore) countForTest() int64 {
+// Count returns the current resource count. It should be used only
```
Comment (strib) [inline kbfssync/semaphore.go]: Drop the 2nd sentence.

Comment (akalin-keybase) [inline kbfssync/semaphore.go]: Done.

**Pull request merged**